src/transformers/models/owlv2/image_processing_owlv2.py

```diff
@@ -588,7 +589,12 @@ def post_process_image_guided_detection(self, outputs, threshold=0.0, nms_thresh
                     scores[idx][ious > nms_threshold] = 0.0
 
         # Convert from relative [0, 1] to absolute [0, height] coordinates
-        img_h, img_w = target_sizes.unbind(1)
+        if target_sizes is not None:
```

Comment:
if `target_sizes` is `None` we still need to set img_h, img_w

Reply:
I thought if `target_sizes` is `None`, then we don't scale the bounding boxes, we leave them in relative coordinates. In which case, we don't need to go anything related to scaling.

Reply:
Right, but the line on L589 expects the img_w and img_h variables, which will now errro out as they're undefined if target_sizes is None. I suspect L598 and L599 should also be indented

Reply:
Yes, they should be inside the if block. That's my mistake.